airmozilla/manage/templates/manage/survey_edit.html

```diff
@@ -40,58 +29,24 @@
     {% include 'manage/_default_form.html' %}
 
     <h3>Questions</h3>
+    <a class="survey-links" href="{{ url('manage:survey_questions', survey.id) }}">Edit questions here</a>
     <table class="table table-striped table-bordered">
       <thead>
         <tr>
           <th>JSON</th>
-          <th>Order</th>
-          <th></th>
         </tr>
       </thead>
       <tbody>
       {% for question in questions %}
       <tr>
           <td class="question">
-            <textarea name="question">{{ json_print(question.question, indent=2) }}</textarea>
-          </td>
-          <td class="ordering">
-            <form method="post" action="{{ url('manage:survey_question_edit', survey.id, question.id) }}">
-              {{ csrf() }}
-              <button class="btn btn-sm ordering-up" name="ordering" value="up"><i class="glyphicon glyphicon-arrow-up"></i></button><br>
-              <button class="btn btn-sm ordering-down" name="ordering" value="down"><i class="glyphicon glyphicon-arrow-down"></i></button>
-            </form>
-          </td>
-          <td>
-            <form method="post" style="display:inline" class="edit"
-                  action="{{ url('manage:survey_question_edit', survey.id, question.id) }}">
-              {{ csrf() }}
-              <button class="btn btn-primary btn-sm">
-                <i class="glyphicon glyphicon-floppy-disk"></i>
-                Save
-              </button>
-            </form>
-            <form method="post" class="confirm"
-                  action="{{ url('manage:survey_question_delete', survey.id, question.id) }}">
-              {{ csrf() }}
-              <button class="btn btn-danger btn-sm" type="submit">
-                <i class="glyphicon glyphicon-trash"></i>
-                Delete
-              </button>
-            </form>
+            <textarea readonly name="question">{{ json_print(question.question, indent=2) }}</textarea>
```

Comment:
If you can't edit it, there's no point this being a `textarea` tag. Use a `<pre>` tag or a plain `<div>` tag and set `overflow:auto` on it.